mediawiki.json

```diff
@@ -1,21 +1,45 @@
 {
-  "extends": [
-    "./vue-es5.js"
-  ],
   "globals": {
     "mw": "readonly",
     "require": "readonly",
     "module": "readonly"
   },
   "plugins": [ "mediawiki" ],
   "rules": {
-    "no-implicit-globals": "off",
     "mediawiki/class-doc": "error",
     "mediawiki/msg-doc": "error",
-    "mediawiki/no-vue-dynamic-i18n": "error",
-    "mediawiki/valid-package-file-require": "error",
-    "vue/v-bind-style": [ "error", "longform" ],
-    "vue/v-slot-style": [ "error", "longform" ],
-    "vue/v-on-style": [ "error", "longform" ]
-  }
+    "mediawiki/valid-package-file-require": "error"
+  },
+  "overrides": [
+    {
+      "files": [ "**/*.vue" ],
+      "extends": [
+        "plugin:vue/recommended",
+        "./vue-es5.js"
+      ],
+      "plugins": [ "es" ],
+      "parserOptions": {
+          "sourceType": "script",
+          "ecmaFeatures": {
+            "jsx": false
+          }
+        },
+        "rules": {
+          "vue/html-indent": [ "warn", "tab" ],
+          "vue/html-closing-bracket-newline": "off",
+          "vue/max-attributes-per-line": [ "warn", {
+            "singleline": 2,
+            "multiline": {
+              "max": 1,
+              "allowFirstLine": true
+            }
+          } ],
```

Comment:
The rules above this point (html-indent, html-closing-bracket-newline, and max-attributes-per-line) should already be inherited from vue-common.json (which vue-es5.js indirectly inherits from)